boot/zephyr/prj.conf

```
@@ -2,18 +2,17 @@ CONFIG_CONSOLE_HANDLER=y
 CONFIG_DEBUG=y
 CONFIG_SYSTEM_CLOCK_DISABLE=y
 CONFIG_SYS_POWER_MANAGEMENT=n
-
 CONFIG_MAIN_STACK_SIZE=10240
 CONFIG_MBEDTLS_CFG_FILE="mcuboot-mbedtls-cfg.h"
```

```suggestion
CONFIG_MBEDTLS_CFG_FILE="config-ecdsa.h"
```
Can we use ECDSA config file

---

We need to keep it generic for key based and X509 based approach hence keeping it generic